dev/temp_norm_annot_patch_wf.cwl

```diff
@@ -0,0 +1,77 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc_norm_annot_wf
+doc: "This is a temp wf to normalize and annotate a PASS vcf"
+requirements:
+  - class: ScatterFeatureRequirement
+  - class: MultipleInputFeatureRequirement
+  - class: SubworkflowFeatureRequirement
+
+inputs:
+  indexed_reference_fasta: {type: File, secondaryFiles: [.fai, ^.dict]}
+  input_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "Input vcf to normalize, annotate, and soft filter"}
+  input_tumor_name: string
+  input_normal_name: string
+  add_common_fields: {type: boolean, doc: "Set to true if input is a strelka2 vcf that hasn't had common fields added", default: false}
+  bcftools_annot_columns: {type: string, doc: "csv string of columns from annotation to port into the input vcf, i.e INFO/AF", default: "INFO/AF"}
+  bcftools_annot_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "bgzipped annotation vcf file"}
+  gatk_filter_name: {type: 'string[]', doc: "Array of names for each filter tag to add, recommend: [\"NORM_DP_LOW\", \"GNOMAD_AF_HIGH\"]"}
+  gatk_filter_expression: {type: 'string[]', doc: "Array of filter expressions to establish criteria to tag variants with. See https://gatk.broadinstitute.org/hc/en-us/articles/360036730071-VariantFiltration, recommend: \"vc.getGenotype('\" + inputs.input_normal_name + \"').getDP() <= 7\"), \"AF > 0.001\"]"}
+  disable_hotspot_annotation: { type: 'boolean?', doc: "Disable Hotspot Annotation and skip this task.", default: false }
+  vep_cache: {type: File, doc: "tar gzipped cache from ensembl/local converted cache"}
+  vep_ref_build: {type: ['null', string], doc: "Genome ref build used, should line up with cache.", default: "GRCh38" }
+  genomic_hotspots: { type: 'File[]?', doc: "Tab-delimited BED formatted file(s) containing hg38 genomic positions corresponding to hotspots" }
+  protein_hotspots: { type: 'File[]?', doc: "Column-name-containing, tab-delimited file(s) containing protein names and HGVSp short values corresponding to hotspots" }
+  output_basename: string
+  tool_name: string
+  retain_info: {type: string?, doc: "csv string with INFO fields that you want to keep, i.e. for consensus `MQ,MQ0,CAL,HotSpotAllele`"}
```

Comment:
I think I don't understand the use case for this, but would it make more sense to input any fields you want to drop, instead of having to figure out what fields are there in order to retain them?